.github/workflows/ci.yml

```diff
@@ -181,6 +183,31 @@ jobs:
         if: ${{ github.ref == 'refs/heads/main'
              || startsWith(github.ref, 'refs/tags/v') }}
 
+  docker:
+    name: Docker image
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: subosito/flutter-action@v2
+        with:
+          flutter-version: ${{ env.FLUTTER_VER }}
+          channel: stable
+          cache: true
+      - uses: docker/setup-buildx-action@v2
+
+      - run: make flutter.pub
```

Comment:
@SleepySquash why do we need it here? We've build everything in the `build` CI job. Just take the artifacts from there and pack into Docker image here.

And I think archivation will shift to the `buidl` stage.

Reply:
@tyranron, ok! It was to speed up the Docker image assembly here, since `build` stage takes about ~10 minutes due to slow Windows compiling and Web only takes about ~2 minutes. But I guess there's really no difference when the image gets built.